bootstrap.sh

```diff
@@ -57,7 +63,8 @@ fi
 # everything under `external/`. That's why include both in the PYTHONPATH
 
 export PATH=$(pwd)/external/usr/bin:$PATH
-export PYTHONPATH=$(pwd)/external:$(pwd)/external/usr/lib/python$pyver/site-packages:$PYTHONPATH
+export PYTHONPATH=$(pwd)/external:$(pwd)/external/${libpath}/python$pyver/site-packages:$PYTHONPATH
```

Comment:
I guess you expect `libpath` to be an absolute path, so I would write this as

```suggestion
export PYTHONPATH=$(pwd)/external:$(pwd)/external${libpath}/python$pyver/site-packages:$PYTHONPATH
```